src/MetricsProvider.ts

```diff
@@ -9,21 +9,24 @@ interface MetricsProviderConstructorOptions {
 }
 
 export default class MetricsProvider {
-  private readonly minimalEvents: eventTypes[] = ['sessions', 'views']
-  private readonly marketingEvents: eventTypes[] = ['attribution', 'users', 'location']
-  private readonly trackingEvents: eventTypes[] = ['events', 'crashes', 'apm']
-  private readonly performanceEvents: eventTypes[] = ['scrolls', 'clicks', 'forms', 'star-rating', 'feedback']
-  private readonly groupedFeatures: Record<consentTypes, eventTypes[]> = {
+  private readonly minimalFeatures: metricFeatures[] = ['sessions', 'views', 'events']
+  private readonly performanceFeatures: metricFeatures[] = ['crashes', 'apm']
+  private readonly uxFeatures: metricFeatures[] = ['scrolls', 'clicks', 'forms']
+  private readonly feedbackFeatures: metricFeatures[] = ['star-rating', 'feedback']
+  private readonly locationFeatures: metricFeatures[] = ['location']
+  private readonly groupedFeatures: Record<consentTypes, metricFeatures[]> = {
     all: [
-      ...this.marketingEvents,
-      ...this.minimalEvents,
-      ...this.performanceEvents,
-      ...this.trackingEvents
+      ...this.minimalFeatures,
+      ...this.locationFeatures,
+      ...this.performanceFeatures,
+      ...this.uxFeatures,
+      ...this.feedbackFeatures
     ],
-    marketing: this.marketingEvents,
-    minimal: this.minimalEvents,
-    performance: this.performanceEvents,
-    tracking: this.trackingEvents
+    minimal: this.minimalFeatures,
+    performance: this.performanceFeatures,
+    ux: this.uxFeatures,
+    feedback: this.feedbackFeatures,
+    location: this.locationFeatures
```

Comment:
nit: do we need locationFeatures if we do not plan to collect this data? unless I am misinterpreting the code it might confuse users if it is here and we are not collecting any info.